flake.nix

```diff
@@ -2,32 +2,19 @@
   description = "Nix CycloneDX Software Bills of Materials (SBOMs)";
 
   inputs = {
-
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
-
     flake-utils.url = "github:numtide/flake-utils";
-
-    flake-compat = {
-      url = "github:edolstra/flake-compat";
-      flake = false;
-    };
-
-    flake-parts = {
-      url = "github:hercules-ci/flake-parts";
-      inputs.nixpkgs-lib.follows = "nixpkgs";
-    };
-
+    flake-compat.url = "github:nix-community/flake-compat";
```

Comment:
The nix-community one seems to be less maintained than eelcos right now.

Reply:
Fixed